examples/upload_artifacts.py

```diff
@@ -0,0 +1,87 @@
+from pycti import OpenCTIApiClient
+import argparse
+import os
+import magic
+
+API_URL = "API_URL_HERE"
+API_TOKEN = "API_TOKEN_HERE"
+
+# OpenCTI instantiation
+OPENCTI_API_CLIENT = OpenCTIApiClient(API_URL, API_TOKEN)
```

Comment:
Please assign the OpenCTIApiClient object to a local variable within the `main` function and hand it over to the `upload` function via an argument.